datalab/README.md

```diff
@@ -0,0 +1,18 @@
+# Detecting issues in an image classification dataset (Caltech-256) with Datalab and a Swin Transformer model.
```

Comment:
Add a reference to this example in the table of contents from the main readme in examples/